cargo-miri/bin.rs

```diff
@@ -495,24 +502,36 @@ fn phase_cargo_rustc(mut args: env::Args) {
     if verbose {
         eprintln!("+ {:?}", cmd);
     }
-    match cmd.status() {
-        Ok(exit) =>
-            if !exit.success() {
-                std::process::exit(exit.code().unwrap_or(42));
-            },
-        Err(e) => panic!("error running {:?}:\n{:?}", cmd, e),
-    }
+    exec(cmd)
 }
 
-fn phase_cargo_runner(binary: &str, args: env::Args) {
-    eprintln!("Asked to execute {}, args: {:?}", binary, args.collect::<Vec<_>>());
+fn phase_cargo_runner(binary: &str, args: env::ArgsOs) {
+    let verbose = std::env::var_os("MIRI_VERBOSE").is_some();
 
     let file = File::open(binary)
         .unwrap_or_else(|_| show_error(format!("File {:?} not found, or cargo-miri invoked incorrectly", binary)));
     let file = BufReader::new(file);
     let info: CrateRunInfo = serde_json::from_reader(file)
         .unwrap_or_else(|_| show_error(format!("File {:?} does not contain valid JSON", binary)));
     // FIXME: remove the file.
+
+    let mut cmd = miri();
+    // Forward rustc arguments,with our sysroot.
```

Comment:
```suggestion
    // Forward rustc arguments, with our sysroot.
```

Reply:
This already got changed in a later commit.